### PR TITLE
Add continuous MyGet package feed

### PR DESCRIPTION
**Pull request by Nihlus**
This PR adds supporting features for a continuous MyGet package feed, built as we work on the `develop` branch. More specifically, it adds a reference to the feed in the README, and a MyGet-specific version override in the build script.

We will also need to add a webhook, so that MyGet is notified of new commits to the selected branch.

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -19,6 +19,9 @@ IF NOT EXIST build.fsx (
 SET BuildTarget=
 if "%BuildRunner%" == "MyGet" (
   SET BuildTarget=NuGet
+
```
Comment (Frassle) [inline build.cmd]: Can you add a comment here in the file about what this is doing? I don't recognise the ### or * syntax (and looking up shell syntax on google is nearly impossible) and have never seen anything called "git build".

Comment (Nihlus) [inline build.cmd]: It's pretty simple - anything after `echo` up to `>` is treated as a string, and is placed into RELEASE_NOTES.md, overwriting it. Then, the same follows, but rather appends an indented description. All it does is overwrite the existing RELEASE_NOTES.md with

```
### <whatever version the package is at
	* git build
```

Comment (Frassle) [inline build.cmd]: :man_facepalming: Right that makes sense

**Comment by Frassle** [comment]
LGTM

**Pull request merged**